src/ca11y/lib/format.js

```diff
@@ -1,5 +1,5 @@
-import strftime from 'fast-strftime'
-
-export default function format(date) {
-  return strftime('%F', date)
+export default function format(state, format, delimiter, date) {
+  const chunks = format.map((format)=> state[format])
```

Comment:
Can omit the parens around the `format` arg in that arrow func. Also :hocho: the log statement?